pkg/cmd/pipeline/start.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/cli/pkg/cmd/pipelinerun"
 	"github.com/tektoncd/cli/pkg/flags"
+	"github.com/tektoncd/cli/pkg/helper/labels"
```

Comment:
nit: I am not a huge fine of the `helper` package but it's already there, so can be changed in a follow-up anyway :wink:

Reply:
This is required for both, `start pipeline` and `start task` so I put this in helper